### PR TITLE
Mheavin doc 636 php windows install

### PR DESCRIPTION
**Pull request by sfc-gh-mheavin**
Update Windows build instructions

DOC-636

### CODE REVIEW

**Review by sfc-gh-spanaite — APPROVED** [review]

**Review by sfc-gh-asaganich — APPROVED** [review]
LGTM!

File: README.rst [hunk 1]
```diff
@@ -87,17 +87,21 @@ Building the Driver on Linux and macOS
...
 **Note**: Snowflake supports only thread-safe versions of PHP.
 
+You must install Microsoft Visual Studio 2019 (VS16) or earlier with the C++ development installer option.
+
 To build the PHP driver for Windows:
 
 #. Download and install the PHP SDK:
 
-    #. Download PHP 8.0.16 binaries from `<https://windows.php.net/downloads/releases/php-8.0.16-Win32-vs16-x64.zip>`_.
+    #. Download the PHP version binaries from `<https://windows.php.net/downloads/releases/>`_, such as `<https://windows.php.net/downloads/releases/php-8.1.18-Win32-vs16-x64.zip>`_.
```
Comment (sfc-gh-ext-simba-hx) [inline README.rst:102]: Better to mention here or somewhere that on Windows NTS is not supported, so do not download binaries with "nts" in the package name.

Comment (sfc-gh-igarish) [reply README.rst:102]: I agree with Harry.

File: README.rst [hunk 2]
```diff
@@ -116,15 +120,15 @@ To build the PHP driver for Windows:
...
      **Do not create this directory.** The script creates this directory for you when downloading the PHP SDK.
 
    For example:
 
    .. parsed-literal::
 
-       .\\scripts\\setup_php_sdk.bat x64 Release |win-vs-version| C:\\php-sdk
+       .\\scripts\\setup_php_sdk.bat x64 Release VS16 C:\\php-sdk
 
 #. Download and install the PHP binaries, or build PHP yourself.
```
Comment (sfc-gh-ext-simba-hx) [inline README.rst:134]: This step is always needed even when downloading PHP binaries directly. To build the driver PHP source code is needed.
So maybe change this step like "download and build PHP source code", and make it mandatory step, not optional.

File: README.rst [hunk 3]
```diff
@@ -87,17 +87,21 @@ Building the Driver on Linux and macOS
...
+.. |win-php-version| replace:: 8.1.18 
+
 
 **Note**: Snowflake supports only thread-safe versions of PHP.
 
+You must install Microsoft Visual Studio 2019 (VS16) or earlier with the C++ development installer option.
+
 To build the PHP driver for Windows:
 
 #. Download and install the PHP SDK:
```
Comment (sfc-gh-ext-simba-hx) [inline README.rst:100]: This step is download and install PHP, not PHP SDK. Remove the option of build PHP from source code and ask customer to always install PHP from official binary would be much easier for us as we don't have to teach customer how to build and install PHP from source code, which kind of out of our scope.

File: README.rst [hunk 4]
```diff
@@ -137,13 +141,13 @@ To build the PHP driver for Windows:
    For :code:`<arch>`, :code:`<build>`, :code:`<visual studio version>`, and :code:`<path to PHP SDK>`, specify the same values
    that you used in the previous step.
 
-   For :code:`<full PHP version>`, specify the full version number of PHP that you want to install (e.g. :code:`7.2.24`).
+   For :code:`<full PHP version>`, specify the full version number of PHP that you want to install (e.g. :code:`8.1.18`).
```
Comment (sfc-gh-ext-simba-hx) [inline README.rst:144]: Change from
"the full version number of PHP that you want to install"
to something like
"the full version number of the PHP binary you installed"

Comment (sfc-gh-mheavin) [reply README.rst:144]: Done

File: README.rst (hunk 1 above)
Comment (sfc-gh-mheavin) [reply README.rst:102]: Done

**Review by sfc-gh-igarish — APPROVED** [review]
LGTM. Please address comments from Harry.

File: README.rst (hunk 3 above)
Comment (sfc-gh-mheavin) [reply README.rst:100]: Done

**Pull request merged**